Gemfile

```diff
@@ -272,3 +272,5 @@ gem 'faraday-retry'
 gem 'net-http'
 
 gem 'webrick', require: false
+
+gem "cgi", ">= 0.3.6", require: false
```

Comment:
@xfalcox would be good to add a note here, so we can clean up once this is fixed upstream.